src/App.js

```diff
@@ -0,0 +1,40 @@
+// import logo from './logo.svg';
+import React from 'react';
+import './App.css';
+// import Layout from './components/Layout';
+import {
+  createBrowserRouter,
+  Route,
+  createRoutesFromElements,
+  RouterProvider,
+} from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Categories from './components/Categories';
+// import {
+//   createBrowserRouter,
+//   Route,
+//   createRoutesFromElements,
+//   RouterProvider,
+// } from 'react-router-dom';
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
Commented piece of code removed. Screen shot attached for your reference
![image](https://user-images.githubusercontent.com/117091923/230486409-dd806183-e2c0-4728-b34c-7c47f6fbeae8.png)